app/views/projects/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="project_show_page">
+  <body>
+    <h1>Project: <%= @project.name %></h1>
+      <p>Material: <%= @project.material %></p>
+      <p>Challenge Theme: <%= @project.challenge.theme %></p>
+      <p>Total Number of Contestants Participating: <%= @project.contestants.count %></p>
+      <p>Average Contestant Experience: <%= @project.contestants.average_xp %> years</p>
```

Comment:
Great job using a model method here. You want to avoid doing database logic in your view 👍 

One things to notice here is that because you made a Contestant class method, you have to call `@project.contestants.average_xp`. There's nothing wrong with this, but the fact that you have to chain two method calls together indicates to me that your model method doesn't quite do exactly what you need. Instead, what I would recommend is to create a Project instance method: `@project.average_xp`.